README.md

```diff
@@ -323,16 +323,19 @@ Alongside the built assets a metadata file will be created with info about the b
         "variant": "hotspot",
         "version": "jdk8u",
         "tag": "jdk8u202-b08",
-        {
+        "version_data": {
             "adopt_build_number": 2,
             "major": 8,
             "minor": 0,
             "security": 202,
             "build": 8,
             "version": "8u202-b08",
             "semver": "8.0.202+8.2",
+            "full_version_output": "<output of java --version>",
+            "configure_args": "<output of bash configure>"
         },
-        "binary_type": "jdk"
+        "binary_type": "jdk",
+        "sha256": "c1b8fb7298d66a5bca9b830e8d612a85bbc52c81b9a88cef4dd71f2f37b289f1"
```

Comment:
perhaps <output of the shasum>

Reply:
Perhaps?

Reply:
Ugh, github was not playing nice today.  Perhaps make that a &lt;sha sam&gt; as opposed to putting in a real value

Reply:
Done